backend/db/utils.py

```diff
@@ -276,6 +276,31 @@ def is_table_up_to_date(table_name: str, skip_if_updated_within_hours: int = Non
     return check_if_updated(last_updated_key, skip_if_updated_within_hours)
 
 
+def reset_refresh_status(local=False):
+    """Reset DB refresh active status variables to reflect non-active status
+
+    Used in situations where it is not active, but the variables reflect otherwise. This can happen due to exiting out
+    of the debugger during a refresh or when a GH action times out."""
+    key_pairs = [
+        ('last_derived_refresh_request', 'last_derived_refresh_exited'),
+        ('last_refresh_request', 'last_refresh_exited')
+    ]
+    for start_time_key, end_time_key in key_pairs:
+        last_start: datetime = dp.parse(check_db_status_var(start_time_key, local))
+        last_end: datetime = dp.parse(check_db_status_var(end_time_key, local))
+        if last_end < last_start:
+            # arbitrarily add 1 sec so considered exited after start
+            last_end: str = (last_start + timedelta(seconds=1)).astimezone(pytz.timezone('America/New_York')).isoformat()
+            update_db_status_var(end_time_key, last_end, local)
+    update_db_status_var('refresh_status', 'inactive', local)
+
+
+def reset_refresh_state(local=False):
```

Comment:
## Add: `reset_refresh_state()`
Meant to only be run manually during development.

Calls: 
- `reset_temp_refresh_tables()`: Function I created last month to clean up the state of temporary tables in case of a failed refresh.
- `reset_refresh_status(local)`: New func I just added in this PR